### DISCUSSION TITLE
how-to-disable-cloudflare-proxying-with-github-pages/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-programming#34] [Comments]
# how-to-disable-cloudflare-proxying-with-github-pages/

When creating a new GitHub pages site with Cloudflare’s DNS, the default setting is proxying all data via Cloudflare. This is OK, but Cloudflare can slow down GitHub pages somewhat, and GitHub pages have no bandwidth limits so might as well serve directly.

https://blog.jakelee.co.uk/how-to-disable-cloudflare-proxying-with-github-pages/

**Comment by Talia04** [comment]
Wow, it worked for me. Thanks! I had visited several sites before, even went to the IT desk at my university, but nothing could fix the issue

**Reply by JakeSteam** [reply]
Happy I could help!
